go.mod

```diff
@@ -29,7 +29,7 @@ require (
 	golang.org/x/tools v0.0.0-20181011042414-1f849cf54d09 // indirect
 	gopkg.in/yaml.v2 v2.2.1
 	k8s.io/gengo v0.0.0-20190128074634-0689ccc1d7d6
-	k8s.io/klog v0.0.0-20181102134211-b9b56d5dfc92
+	k8s.io/klog v0.0.0-20181102134211-b9b56d5dfc92 // indirect
```

Comment:
Thanks for your contribution! As https://github.com/kubernetes/kube-openapi/pull/134#issuecomment-459689388 pointed out, klog is a transitive dependency from gengo. Without changing gengo, this does help us get rid of the dep

/close